activation.md

```diff
@@ -1,4 +1,28 @@
 ---
 description: >-
   Descibes the licencing of SysKit Sense and how to activate the solution.
----
\ No newline at end of file
+---
+

```

Comment:
We need an title for the article, Licensing and Activation maybe?

Reply:
renamed the file. check the new one for the change.